cmd/activate.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2022-2023 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/google/uuid"
+	"github.com/spf13/cobra"
+)
+
+var (
+	activateCmd = &cobra.Command{
+		Use:   "activate SCHEME UUID",
+		Short: "activate policy UUID for scheme SCHEME",
```

Comment:
Please also consider Long format as used for `evcli` and `cocli` tools, those have been found very useful for external audience